web/src/topic_list.ts

```diff
@@ -317,15 +348,20 @@ export function initialize({on_topic_click}) {
             }
 
             const $stream_row = $(e.target).parents(".narrow-filter");
-            const stream_id = Number.parseInt($stream_row.attr("data-stream-id"), 10);
+            const stream_id_string = $stream_row.attr("data-stream-id");
+            assert(stream_id_string !== undefined);
+            const stream_id = Number.parseInt(stream_id_string, 10);
             const topic = $(e.target).parents("li").attr("data-topic-name");
             on_topic_click(stream_id, topic);
 
             e.preventDefault();
         },
     );
 
-    $("body").on("input", "#filter-topic-input", () => {
-        active_widgets.get(active_stream_id()).build();
+    $("body").on("input", "#filter-topic-input", (): void => {
+        const stream_id = active_stream_id();
+        if (stream_id !== undefined) {
+            active_widgets.get(stream_id)?.build();
+        }
```

Comment:
Would an assert be more appropriate here? I'm pretty sure `filter-topics-input` isn't supposed to exist if no widget is active.

Reply:
Yeah I guess this would have thrown an error before if it was `undefined`, so it's reasonable to assert here.